lesson1.md

```diff
@@ -122,6 +120,15 @@ We'll be using these tools during our course:
 
 ---
 
+# The standard
+
+#### http://www.ecma-international.org/ecma-262/10.0/index.html
+
+* Standardized Specification of JavaScript
+* Very technical, not good for learning
+
+---
+
 ### Special Characters
```

Comment:
This seems out of place

Reply:
It's right under "additional courseware", I think we shouldn't have a JavaScript course without referring to THE STANDARD at least once :)